docs/menu.yml

```diff
@@ -36,14 +36,22 @@ catalog:
         path: "/en/setup/service-agent/java-agent/token-auth"
       - name: "Manual APIs"
         catalog:
-          - name: "Tracing APIs"
```

Comment:
This should be kept, and all following news as a catalog(sub level)

Reply:
Ok